src/index.ts

```diff
@@ -65,38 +105,113 @@ export async function run(): Promise<void> {
     await execute(podman, args);
 
     core.info(`Successfully pushed ${imageToPush} to ${registryPath}.`);
-    core.setOutput('registry-path', registryPath);
+    core.setOutput("registry-path", registryPath);
+
+    // remove the pulled image from the Podman local registry
+    if (isPushingDockerImage) {
+        core.info(`Removing ${imageToPush} from the Podman local registry`);
+        await execute(podman, [ "rmi", imageToPush ]);
+    }
 
     try {
         const digest = (await fs.promises.readFile(digestFile)).toString();
         core.info(digest);
-        core.setOutput('digest', digest);
+        core.setOutput("digest", digest);
     }
     catch (err) {
         core.warning(`Failed to read digest file "${digestFile}": ${err}`);
     }
 }
 
-async function execute(executable: string, args: string[], execOptions: exec.ExecOptions = {}): Promise<{ exitCode: number, stdout: string, stderr: string }> {
+async function pullImageFromDocker(
+    imageName: string,
+    podman: string,
+): Promise<boolean> {
+    try {
+        await execute(podman, [ "pull", `docker-daemon:${imageName}` ]);
+        core.info("Image found and sucessfully pulled from Docker local registry");
+        return true;
+    }
+    catch (err) {
+        core.info("Image not found in Docker local registry");
```

Comment:
Log the error

Reply:
Logging the error here will something look like `error` to the users(and it is not something error) as it will have "error" word written all over the logs.
What do you think?